src/it/java/net/spy/memcached/ClientBaseCase.java

```diff
@@ -151,6 +160,29 @@ protected void restoreClusterConfig(String config, Collection<NodeEndPoint> endp
 	}
   }
 
+  protected void restoreClusterConfigForTLS(int portNumber) throws Exception {
+      List<InetSocketAddress> addrs = AddrUtil.getAddresses(TestConfig.IPV4_ADDR
+      + ":" + portNumber);
+
+      MemcachedClient client = new MemcachedClient(new ClientTestConnectionFactory(){
+        @Override
+        public ClientMode getClientMode() {
+          return ClientMode.Static;
+        }
+      }, addrs);
+      client.set(ConfigurationType.CLUSTER.getValueWithNameSpace(), 0, "1\n" + "localhost.localdomain|" + TestConfig.IPV4_ADDR + "|" + portNumber);
+      Thread.sleep(1000); // wait for all configurations to apply
+  }
+
+  protected MemcachedClient staticMemcachedClient(List<InetSocketAddress> addrs) throws IOException {
```

Comment:
Probably you can make this method static so you can also use it from `AutoDiscoveryTest.java`.

Reply:
Will do